corehq/apps/cloudcare/static/cloudcare/js/formplayer/app.js

```diff
@@ -51,6 +75,7 @@ hqDefine("cloudcare/js/formplayer/app", function () {
         });
     });
 
+    // TODO: navigation isn't navigating
```

Comment:
When will this be resolved? If it is in a later commit, ignore this.

Reply:
yep, later commit

Reply:
There is still one TODO, in a test file, that I'm intending to fix.